source/kraken/tests/apply_disruption_test.cpp

```diff
@@ -1357,6 +1357,160 @@ BOOST_AUTO_TEST_CASE(add_simple_impact_on_line_section) {
     BOOST_REQUIRE_EQUAL(vj->stop_time_list.back().stop_point->uri, "stop_point:40");
 }
 
+BOOST_AUTO_TEST_CASE(multiple_impact_on_line_section) {
```

Comment:
Nice test, an ASCII art would be good (almost like the one in this PR description), but **minor**.